_posts/2015-06-03-team-offsite.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Team ePages 6 offsite event"
+date: "2015-06-03 13:44:00"
+icon: users
+tags: team event offsite
+categories: events
+authors: ["Fouad-Steffen Nabhan"]
+---
+
+Working hard and embrace the nature? How does that fit together? Our ePages 6 development team organised a team offsite event and convened in a lovely castle to focus on knowledge transfer. What did they do? Read on!
+
+We found this nice place called "Die Burg Schöna" right in the last corner of Saxon Switzerland, Germany to come together some hours away from the office to approach a tight agenda of topics in our team.
+
+Surrounded by nature...
+
+{% image blog-offsite-landscape.jpg %}{% endimage %}
+
+...we discovered the benefits of team work on more than one occasion. It helped find some well hidden caches, to manage some obstacles during our team event as well as to get the information we needed to flow freely amongst all team members.
```

Comment:
I agree with Kristof, internally this is fine - we all know what is meant, but externally this portrays an image that we had no team work before and we could not communicate properly. Here are my suggestions.

"We had to utilise and build upon our team work on more than one occasion, in order to overcome some obstacles, steep mountains as well as find some hidden geo caches. This pulled us together as a unit even more than previously."